server/app.py

```diff
@@ -1,19 +1,23 @@
-from flask import Flask 
-from config import Config 
+from api.ping_handler import ping_handler
+from api.home_handler import home_handler
+from flask import Flask
+from config import Config
 from flask_sqlalchemy import SQLAlchemy
 from flask_migrate import Migrate
+from flask_cors import CORS
 
-app = Flask(__name__) # ProductLaunch app
+app = Flask(__name__)  # ProductLaunch app
 app.config.from_object(Config)
 
-db = SQLAlchemy(app) 
+# Allows CORS on all routes (for development convenience). TODO: CORS config
+# should be changed for production.
+cors = CORS(app)
+
+db = SQLAlchemy(app)
 migrate = Migrate(app, db)
 
 # for flask-migrate to generate migration scripts
-from db_models import pluser
```

Comment:
Is there a reason you removed this model import?

Reply:
It was giving me a circular import error, as the app module was importing pluser, and the pluser module was trying to import the app module. I've realized that this was an issue with VS Code's formatting bringing the import to the top of the file. I've re-added the import with a new commit.